spec/rubocop/cop/rspec/variable_name_spec.rb

```diff
@@ -188,4 +188,33 @@
       end
     end
   end
+
+  context 'when configured to ignore certain patterns' do
+    context 'when the variable does not match any of the patterns' do
+      let(:cop_config) do
+        { 'EnforcedStyle' => 'snake_case',
+          'IgnoredPatterns' => ['^userFood$', '^userPet$'] }
+      end
+
+      it 'registers an offense' do
```

Comment:
Sorry for nitpicking, but I’ll suggest removing the innermost `context`s, and use the same `cop_config` for these two examples. E.g.:
```rb
context 'when configured to ignore certain patterhs' do
  let(:cop_config) do
    # …
  end

  it 'registers an offense when not matching any ignored pattern' do
    # …
  end

  it 'registers an offense when matching at least one ignored pattern' do
    # …
  end
end
```

What do you think?